Fail to generate compose file with delayed-start parameter

<!--🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅

Hello there! 😄

To expedite issue processing please search open and closed issues before submitting a new one.
Existing issues often contain information about workarounds, resolution, or progress updates.

Please do not remove sections of this template when submitting your issue.  If the section is not relevant to your issue, leave the section but enter "n/a" for not applicable.

Note some sections are marked "REQUIRED" and must be provided.  Never mark required sections "n/a".

🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅🔅-->


# 🐞 Bug Report

### Description and Minimal Reproduction [**REQUIRED**]
$ make gen delayed-start


## 🔥 Exception or Error
<pre><code>
echo MQTT_VERBOSE=
MQTT_VERBOSE=
docker compose  -p edgex -f docker-compose-base.yml -f ./gen_ext_compose/add-support-notifications-runtime-token-config.yml -f  -f add-delayed-start-services.yml -f add-security.yml -f add-secure-postgres.yml -f add-mqtt-messagebus.yml -f add-secure-mqtt-messagebus.yml -f add-mqtt-broker-mosquitto.yml -f add-security-proxy.yml convert --output docker-compose.yml.gen
open /home/cherry/edgexfoundry/edgex-compose/compose-builder/-f: no such file or directory
make: *** [Makefile:1099: gen] Error 14

</code></pre>


## 🌍  Your Environment
**Deployment Environment:**

**EdgeX Version [**REQUIRED**]:**


**Anything else relevant?**

--- a/compose-builder/Makefile
+++ b/compose-builder/Makefile
@@ -140,6 +140,7 @@ ifeq (delayed-start,$(filter delayed-start,$(ARGS)))
 
     # Generate runtime token config for support services
     ext_file_sup_notif := $(shell ZERO_TRUST="$(MAKE_ZERO_TRUST)" GEN_EXT_DIR="$(GEN_EXT_DIR)" ./gen_runtime_token_config_compose_ext.sh support-notifications)
+	ext_file_sup_sch := $(shell ZERO_TRUST="$(MAKE_ZERO_TRUST)" GEN_EXT_DIR="$(GEN_EXT_DIR)"  ./gen_runtime_token_config_compose_ext.sh support-scheduler)
 
     # Add generated config files to COMPOSE_FILES
     COMPOSE_FILES += -f $(ext_file_sup_notif) -f $(ext_file_sup_sch)

